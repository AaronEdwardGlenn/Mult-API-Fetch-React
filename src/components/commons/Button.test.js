import React from 'react';
import { shallow } from 'enzyme'
;
import Button from './Button';

describe('our button component', () => {
  it('can match the snap', () => {
    const wrapper = shallow(<Button text="BUTT" />);
    expect(wrapper).toMatchSnapshot();
    
  });
  it('can match the snap with a prop passed', () => {
    const wrapper = shallow(<Button text="COOL" value="BUTT" />);
    expect(wrapper).toMatchSnapshot();
  }); 
})
;
