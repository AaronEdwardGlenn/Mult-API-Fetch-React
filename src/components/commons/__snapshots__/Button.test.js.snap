// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`our button component can match the snap 1`] = `
<button>
  BUTT
</button>
`;

exports[`our button component can match the snap with a prop passed 1`] = `
<button
  value="BUTT"
>
  COOL
</button>
`;
